python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1042,10 +1042,39 @@ def execute_asset_backfill_iteration(
             )
             instance.update_backfill(updated_backfill)
 
+        new_materialized_partitions = (
+            updated_backfill_data.materialized_subset
+            - previous_asset_backfill_data.materialized_subset
+        )
+        new_failed_partitions = (
+            updated_backfill_data.failed_and_downstream_subset
+            - previous_asset_backfill_data.failed_and_downstream_subset
+        )
+        updated_backfill_in_progress = (
+            updated_backfill_data.requested_subset - updated_backfill_data.materialized_subset
+        )
+        previous_backfill_in_progress = (
+            previous_asset_backfill_data.requested_subset
+            - previous_asset_backfill_data.materialized_subset
+        )
+        new_requested_partitions = updated_backfill_in_progress - previous_backfill_in_progress
         logger.info(
             f"Asset backfill {updated_backfill.backfill_id} completed iteration with status {updated_backfill.status}."
         )
         logger.info(
+            "Backfill iteration summary:\n"
+            f"**Assets materialized since last iteration:**\n {_asset_key_partition_key_iter_to_str(new_materialized_partitions.iterate_asset_partitions()) if new_materialized_partitions.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
```

Comment:
Should "assets" instead be "asset partitions" in these logs?

Reply:
it could also be unpartitioned assets, so i just went with assets in the more generic sense. I'm fine with asset partitions too! i figured "assets" when the assets were partitioned made more sense that "asset partitions" if one of the assets is unpartitioned, but that's not strongly held